### ISSUE TITLE
feature-homepage-launch branch: Fix projects' info pages' styling of the header

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3377]
### Overview
As developers, we need to fix the styling of the first header in the projects' info pages in the feature-homepage-launch branch so that it matches the styling of those pages on the live website. 

### Details
<details>
  <summary>Click here to see examples of what the header should look like on the current projects' info pages</summary>
  <img width="960" alt="100Automations" src="https://user-images.githubusercontent.com/31293603/189732474-90a685f6-716e-4637-b906-d6d5ad9f464d.png">

  <img width="960" alt="Home Unite Us" src="https://user-images.githubusercontent.com/31293603/189732468-cc564865-f8d1-4b0f-b3b0-b415f55c38d6.png">

  <img width="960" alt="Spare" src="https://user-images.githubusercontent.com/31293603/189732473-2230586d-ebe7-44e7-8849-81fa517f4397.png">
</details>

<details>
  <summary>Click here to see examples of what the header looks like on the feature-homepage-launch projects' info pages</summary>
  <img width="960" alt="100 Automations" src="https://user-images.githubusercontent.com/31293603/189732716-f0612b0a-44b1-4a77-ba0f-08fb303ddf34.png">

  <img width="960" alt="Home Unite Us" src="https://user-images.githubusercontent.com/31293603/189732718-666441cc-d842-488d-8083-938101049ab4.png">

  <img width="960" alt="Spare" src="https://user-images.githubusercontent.com/31293603/189732714-b399317b-a83f-489b-93b8-6efcc1ad79b4.png">

</details>

On the feature-homepage-launch branch, the problem crept in on the project info pages when the SCSS class `hero-inner` was removed from home.scss as part of pr #2373, which was the initial pr on the feature-homepage-launch branch. I think the problem happened because a lot of the code originally in home.scss was removed/changed in order to style the redesigned homepage, including the removal of class `hero-inner`.

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] Familiarize yourself with how the headers look on the projects' info pages on the current website. See the Details section above for screenshots of examples; in addition, here are links to those example pages:
  - [x] https://www.hackforla.org/projects/100-automations
  - [x] https://www.hackforla.org/projects/home-unite-us
  - [x] https://www.hackforla.org/projects/spare
- [x] Fix the SCSS for the header on the projects' info pages on the feature-homepage-launch branch to match how it currently looks on the website by doing the following:
  - [x] In VS Code or your local IDE on your computer, navigate to `_sass/components/_project-page.scss`
  - [x] Then, add the following SCSS code:
    ```
    .hero-inner {
      color: $color-white;
      margin: 60px auto 0px;
      max-width: $max-content-width;
      position: relative;
      z-index: 2;
    
      @media #{$bp-tablet-up} {
        margin-bottom: 0;
        margin-top: 150px;
      }
    
      h1 {
        font-weight: bold;
        margin-bottom: .65rem;
      }
      
      p {
        margin-bottom: .65rem;
      }
      
      span {
        border-bottom: 3px solid $color-red;
      }
    }
    ```
- [x] Using Docker, make sure any changes you make to the code doesn't break anything on the website in mobile, tablet, and desktop views. 
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
  
### Resources/Instructions
- Tracked in https://github.com/hackforla/website/issues/2481
- [How to work off of a feature branch wiki page](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [_sass/components/_project-page.scss](https://github.com/hackforla/website/blob/feature-homepage-launch/_sass/components/_project-page.scss)

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng what should we do with this issue?

**Comment by JessicaLucindaCheng** [comment]
> @JessicaLucindaCheng what should we do with this issue?

@ExperimentsInHonesty I updated the issue. It is ready for you to review and add a milestone if you think it is ready for it.

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng I am confused, how did a problem creep in on the project info pages?  Why did they change at all?  What do they have to do with the Homepage launch?

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty

> I am confused, how did a problem creep in on the project info pages?

The problem crept in on the project info pages when the SCSS class `hero-inner` was removed from home.scss as part of pr #2373, which was the initial pr on the feature-homepage-launch branch.


> Why did they change at all?

I think it changed/problem happened because a lot of the code originally in home.scss was removed/changed in order to style the redesigned homepage, including the removal of class `hero-inner`. 


> What do they have to do with the Homepage launch?

The problem had to do with the Homepage launch because
- the problem only occurs on the feature-homepage-launch branch and not on the gh-pages branch
- and it happened because of the removal of the `hero-inner` class from home.scss, when a lot of the code in home.scss was changed/removed in order to style the redesigned homepage.

Thus, I thought it would be best to address the problem in the feature-homepage-launch branch since that is where the problem is visible.



Also, I have updated the issue with more details about the problem. I am adding the `ready for milestone` back on because I am not sure what label to add to let you or other PMs know to take a look at the issue again.
